<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelTabFrameWidget</class>
 <widget class="QWidget">
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Encoding:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="frameTypeComboBox">
       <property name="minimumWidth">
        <number>160</number>
       </property>
       <property name="maximumWidth">
        <number>160</number>
       </property>
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Transparent square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Left Triangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right Triangle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Left Trapezoid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right Trapezoid</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frameTypeMsgLabel"/>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
